templates/latex/doxygen.sty

```diff
@@ -334,7 +334,7 @@
     \hline%
     \endhead%
 }{%
-    \end{longtabu*}%
+    \end{longtable}%
```

Comment:
What is the reason for changing here from longtabu to longtable, the longtabu tables look much better and are more flexible in respect to log / small parameter names (this said although it is known that longtabu is unmaintained, see https://github.com/tabu-issues-for-future-maintainer/tabu but we have found a reasonable workaround for this)

Reply:
Longtabu ist broken in group definitions. I know, you fixed it, but it seems, that it is still borken in this circumstances. I could not get it working for my external dependencies ([FreeRTOS](https://github.com/FreeRTOS/FreeRTOS-Kernel), [STM32CubeF4](https://github.com/STMicroelectronics/STM32CubeF4)) with longtabu.

Reply:
I cloned the FreeRTOS repository, created a default Doxyfile with just `RECURSIVE=YES` (as instructed in #7816), ran doxygen and ran make in the latex directory and got the same problem.

I'll have to look into this issue.

Reply:
I know, but the workaround does not work for me. I tried to create a simple example, but the error does not show up.

Reply:
See the comment https://github.com/doxygen/doxygen/issues/7816#issuecomment-636332309 indicating incorrect usage of `<pre>` tags. Resolving the real problem properly resolves the issue with tabu.